src/ComputeJoinPoint/index.ts

```diff
@@ -0,0 +1,49 @@
+import { IMinAndMax } from './interface';
+
+export class ComputeJoinPoint {
+  private getMinAndMax(num1: number, num2: number): IMinAndMax {
+    if (num1 < num2)
+      return {
+        min: num1,
+        max: num2
+      };
+    return {
+      min: num2,
+      max: num1
+    };
+  }
+
+  private addition(num: number) {
+    return num
+      .toString()
+      .split('')
+      .reduce((acc: number, current: string) => {
+        return acc + parseInt(current);
+      }, num);
+  }
+
+  static get(num1: number, num2: number): number {
+    if (num1 === num2) return num1;
+    const compute = new ComputeJoinPoint();
+
+    const numbers = compute.getMinAndMax(num1, num2);
```

Comment:
You can just use let, so later you do not have to declare `sumOfMin ` and `sumOfMax `

```suggestion
    let {min, max} = compute.getMinAndMax(num1, num2);
```